plutus-core/common/Annotation.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE TypeFamilies   #-}
+-- GHC 8.10 wans about the derived MonoFoldable instance, GHC>=9.2 works fine
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+module Annotation
```

Comment:
I'd prefer prefixing the module name with something, either `PlutusCore.Annotation` or `Plutus.Annotation` or `PlutusCore.Common.Annotation` etc. This makes it easy to tell that it is a local module from this repo.